api/v1/openapi.yaml

```diff
@@ -249,6 +249,29 @@ paths:
           description: Endpoints with provided parameters not found
         '429':
           description: Rate-limiting too many requests in the given time frame
+    delete:
```

Comment:
Can we rename this to `deleteList` to make the intent more clear? There is already a `delete` under `endpoint/id` api - https://github.com/aditighag/cilium/blob/master/api/v1/openapi.yaml#L170.

Reply:
The `delete` here refers to the `DELETE` HTTP request method, so I don't think we can change this.